tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "compilerOptions": {
+    "lib": [
+      "esnext",
+      "deno.window",
+    ]
+  }
+}
```

Comment:
I believe this is covered by the implied Deno tsconfig. https://deno.land/manual@v1.28.3/advanced/typescript/configuration#what-an-implied-tsconfigjson-looks-like

Reply:
True. Thanks.